docs/using/subscriptions.md

```diff
@@ -0,0 +1,25 @@
+# Subscriptions in HyP3
+For monitoring applications, we provide a subscription service for On Demand processing, which automatically processes new Sentinel-1 acquisitions as they appear in ASF's data holdings. 
+
+Users define their area of interest and search parameters, along with On Demand processing types and options, and set a date range that extends into the future. As ASF receives new acquisitions that meet the search criteria for the subscription, they will automatically be submitted to HyP3 for On Demand processing. 
+
+This service alleviates the need for users to search through acquisitions manually to discover any new images to process for their area of interest. It is particularly useful for ongoing monitoring efforts where users need regular updates over an area of interest, and want to make sure that they process all available images for that area.
+
+## Guidelines for Using Subscriptions
+1. Subscriptions are only necessary for applications that require imagery to be processed beyond the date of the initial search. If all of the images required for a project were acquired before the date of the search, they can be submitted directly for [On Demand processing](https://search.asf.alaska.edu/#/?topic=onDemand){target=_blank}.
+   
+2. Set an end date that is appropriate for your use case. You must define an end date; you cannot set an open-ended subscription. If your project extends beyond your original end date, you can change it, so there is no need to set it far into the future.
+   
+3. Set your AOI to be as small and targeted as possible to minimize the number of granules submitted for processing. Consider applying additional filters to ensure that you are only processing the images you really want; it can be easy to exceed your processing quota if your parameters are set too broadly.  
+   
+4. On Demand products are deleted from storage 14 days after they are generated. Make sure to download the products generated by your subscription on a regular basis so that they don't expire before you access them. 
+
+5. Deactivate your subscription when you no longer need it. If you want to restart your monitoring effort later, you can start a new subscription with a start date that will fill the gap if necessary.
```

Comment:
@jhkennedy Does [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/165/commits/9aaecb8ad0cb06f430e83c8e373674fa1804858e) accurately describe the behavior? 
![image](https://user-images.githubusercontent.com/61886203/135698445-1e480067-2d7e-4b48-9def-2266fc6010f3.png)

Reply:
@hjkristenson I think that covers it (better than what I was working on). The only caveat being, if they disable *before the end date* and re-enable *after the end date* (so there was unprocessed stuff, but now the end date is long passed), I don't know if it'll be evaluated or not. @Jlrine2 @asjohnston-asf ?

Reply:
Oh, I see. I was thinking that it would fill it in, even if time had gone by. That seems a bit dangerous, so I'm sort of relieved to hear that may not be the way it works. I'll wait to hear from those in the know before amending further, but my current understanding is that:
- they can reactivate the subscription and fill in the gap if the original end date has not yet passed
- if the end date of the original subscription has passed, they could create a new subscription and set a start date that would encompass the gap
- if they don't need the stuff in between, they can just start a new subscription (but could re-use the same project name if desired)

Reply:
:+1: those details may evolve anyways so a little vague there until we have the UI up and running and SDK support is good